odoo_install.sh

```diff
@@ -244,6 +255,88 @@ sudo chown root: /etc/init.d/$OE_CONFIG
 echo -e "* Start ODOO on Startup"
 sudo update-rc.d $OE_CONFIG defaults
 
+#--------------------------------------------------
+# Install Nginx if needed
+#--------------------------------------------------
+if [ $INSTALL_NGINX = "True" ]; then
+  echo -e "\n---- Installing and setting up Nginx ----"
+  sudo apt install nginx -y
+  cat <<EOF > ~/$OE_USER
+   server {
+   listen 80;
+
+   # set proper server name after domain set
+   server_name $WEBSITE_NAME;
+
+   # Add Headers for odoo proxy mode
+   proxy_set_header X-Forwarded-Host \$host;
+   proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
+   proxy_set_header X-Forwarded-Proto \$scheme;
+   proxy_set_header X-Real-IP \$remote_addr;
```

Comment:
Maybe we should add these two too?
 proxy_set_header X-Client-IP $remote_addr;
 proxy_set_header HTTP_X_FORWARDED_HOST $remote_addr;